python/hail/dataset.py

```diff
@@ -2083,23 +1976,10 @@ def impute_sex(self, maf_threshold=0.0, include_par=False, female_threshold=0.2,
         :rtype: :py:class:`.VariantDataset`
         """
 
-        pargs = ['imputesex']
-        if maf_threshold:
-            pargs.append('--maf-threshold')
-            pargs.append(str(maf_threshold))
-        if include_par:
-            pargs.append('--include_par')
-        if female_threshold:
-            pargs.append('--female-threshold')
-            pargs.append(str(female_threshold))
-        if male_threshold:
-            pargs.append('--male-threshold')
-            pargs.append(str(male_threshold))
-        if pop_freq:
-            pargs.append('--pop-freq')
-            pargs.append(pop_freq)
-        return self.hc._run_command(self, pargs)
+        jvds = self._jvdf.imputeSex(maf_threshold, include_par, female_threshold, male_threshold, joption(pop_freq))
```

Comment:
I assigned where it looked to long to do it without an assignment.  I'd like to leave it that way, if you're OK with that.